Fabric.Identity.API/Models/ModelExtensions.cs

```diff
@@ -76,7 +78,7 @@ public static IS4.Client ToIs4ClientModel(this Client client)
                     Claims = client.Claims,
                     AlwaysSendClientClaims = client.AlwaysSendClientClaims,
                     AlwaysIncludeUserClaimsInIdToken = client.AlwaysIncludeUserClaimsInIdToken,
-                    PrefixClientClaims = client.PrefixClientClaims,
+                    ClientClaimsPrefix = client.PrefixClientClaims ? "client_" : null,

```

Comment:
Was the prefixclientclaims set to "client_" before? Is this just to distinguish client claims from user claims?